net/instaweb/rewriter/rewrite_options.cc

```diff
@@ -995,6 +996,9 @@ std::vector<DeprecatedOptionMap> kDeprecatedOptionNameList(
     kDeprecatedOptionNameData + arraysize(kDeprecatedOptionNameData)
 );
 
+int kNonAddableResourceHeadersSize = 12;
+StringPieceVector kNonAddableResourceHeaders(kNonAddableResourceHeadersSize);
```

Comment:
kDeprecatedOptionNameList is definitely not valid.  Sorry about the
confusion; that slipped through the review process.


On Thu, Dec 10, 2015 at 4:22 AM, Otto van der Schaaf <
notifications@github.com> wrote:

> In net/instaweb/rewriter/rewrite_options.cc
> <https://github.com/pagespeed/mod_pagespeed/pull/1200#discussion_r47203842>
> :
>
> > @@ -995,6 +996,9 @@ std::vector<DeprecatedOptionMap> kDeprecatedOptionNameList(
> >      kDeprecatedOptionNameData + arraysize(kDeprecatedOptionNameData)
> >  );
> >
> > +int kNonAddableResourceHeadersSize = 12;
> > +StringPieceVector kNonAddableResourceHeaders(kNonAddableResourceHeadersSize);
>
> @jmarantz <https://github.com/jmarantz> Out of curiosity, is
> kDeprecatedOptionNameList (declared one line above) valid according to the
> style guide? That line was why I thought that what I had initially was OK.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pagespeed/mod_pagespeed/pull/1200/files#r47203842>.
>